editor/editor_help.cpp

```diff
@@ -686,7 +668,7 @@ void EditorHelp::_pop_code_font() {
 
 Error EditorHelp::_goto_desc(const String &p_class) {
 	// If class doesn't have docs listed, attempt on-demand docgen
```

Comment:
```suggestion
	// If class doesn't have docs listed.
```
Or completely remove the comment, it does not gives any useful information.